src/theme/as24/colors.ts

```diff
@@ -16,6 +18,7 @@ const colors: Colors = {
     700: '#665E00',
     800: '#4C4500',
     900: '#3A3400',
+    primary: brandPrimary,
```

Comment:
Alternatively, we can leverage [semantic tokens from chakra](https://chakra-ui.com/docs/styled-system/features/semantic-tokens#theme-example) for naming certain values. I wasn't aware of it's existence when implementing the PoC

Reply:
as we discussed, we will bring the semantic tokens separately